### PR TITLE
Removes APEL Server from APEL REST Docker image and adds file for docker-compose deployment

### PR DESCRIPTION
**Pull request by gregcorbett**
This PR removes the apel server from this docker build, requiring the launch of a separate container for the APEL server.

- it removes the conf files and scripts that were only included for the server
- it removes server only requirement (python-daemon)
- alters on entry configuration of docker image, as conf files are now mounted into the REST container
- adds configuration files, previously created on entry, to the git repo
- moves existing conf files / scripts needed for docker build to `docker/`
- renames `cloud-extra.sql` to `extra-cloud.sql` as the new deployment method deploys schema files in alphabetical order
- adds `docker-compose.yaml` and env files so the system can be deployed with docker-compose
- updates documentation to reflect new `docker-compose` deployment

### CODE REVIEW

**Comment by tofu-rocketry** [comment]
This is quite a large PR that seems to mix at least two, possibly three, features together (removal of APEL server, docker-compose stuff, and maybe config changes). Can it be split into separate pull requests using a reasonable amount of effort? From a glance, the commits seem to be separate at least.

A branch, which should end in a PR, should be a single feature to make the branch history clearer and for ease of reviewing. Also, need more context and "why"s in PRs and commit messages. I think I agree with most of the stuff in [this guide](https://chris.beams.io/posts/git-commit/).

**Comment by gregcorbett** [comment]
I have split the first commit up and compressed the removal of file commits.

I've also tried to add why the changes have been made in the commit messages

**Comment by gregcorbett** [comment]
The other options is a docker image with the schema files pre loaded/empty database pre created, but I'm not sure that's better because

- It means we have another docker image to support
- All the schemas you want to have in the image to be known at build time. Then if you wanted a subset of the databases you'd be stuck, you'd just have to ignore the unused databases
- I think it would involve a git repo for just the schema files so the docker could be built automatically

I agree the alphabetical method seems quite fragile but I think it's the best of the two options, I'll add numbers and a README.md documenting this feature

I'll make the documentation changes as well

File: doc/admin.md [hunk 1]
```diff
@@ -71,11 +77,11 @@ You should now have terminal access to the Accounting Server.
 
 3. Click Save.
 
-4. Store the ClientID, Client Secret, and Registration Access Token; as the ID and Secret will need to be put in `apel_rest/settings.py`, and the token will be needed to make further modifications to this registration.
+4. Store the ClientID, Client Secret, and Registration Access Token; as the ID and Secret will need to be put in `yaml/accounting-rest-interface-rc.yaml`, and the token will be needed to make further modifications to this registration.
 
 ## Authorize new WP5 components to view Summaries
```
Comment (gregcorbett) [inline doc/admin.md]: done

File: doc/service_reference_card.md [hunk 2]
```diff
@@ -1,26 +1,44 @@
 # Accounting Service Reference Card
 
+The Accounting service is made up of three containers
+* APEL REST Interface
+* APEL Server
+* MySQL Database [Standard MySQL:5.6 image](https://hub.docker.com/_/mysql/) 
+
 * Daemons running
```
Comment (gregcorbett) [inline doc/service_reference_card.md]: done

File: schemas/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Schema Naming Conventions
+The schema files here are pre-fixed with a numeral to insure their alphanumeric order is the same order that the schema files should be applied to a fresh MySQL installation / container.
```
Comment (tofu-rocketry) [inline schemas/README.md]: "ensure"

**Comment by gregcorbett** [comment]
grammar change made

**Comment by tofu-rocketry** [comment]
Travis has run, but test results don't seem to be getting back.

**Comment by gregcorbett** [comment]
yeah... maybe when I do the rebase it will trick travis into updating

**Comment by tofu-rocketry** [comment]
This needs rebasing and the conflicts resolving. Need any help with that?

**Comment by gregcorbett** [comment]
I should be able to do the rebase/handle the merge conflicts. It's waiting on the APEL server docker image [PR](https://github.com/apel/apel/pull/124).

**Comment by gregcorbett** [comment]
No longer dependant on apel [PR#124](https://github.com/apel/apel/pull/124) due to that functionality being added to the Indigo DataCloud Fork of APEL (see [PR#1](https://github.com/indigo-dc/apel/pull/1)).

I'll rebase this branch and fix the conflicts.

**Comment by gregcorbett** [comment]
This has been rebased and is ready for merging, but the rebase dismissed your review.

**Pull request merged**